docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.6'
+services:
+  '7.2': &base
+    image: "circleci/php:7.2"
+    ulimits:
+      core: 99999999999
+    working_dir: '/home/circleci/app'
+    stdin_open: true
+    tty: true
+    volumes: [ '.:/home/circleci/app' ]
+    tmpfs: [ '/home/circleci/app/tmp:uid=3434,gid=3434,exec' ]
+    depends_on:
+      - redis_integration
+      - ddagent_integration
+    environment:
+      - REDIS_HOSTNAME=redis_integration
+      - DDAGENT_HOSTNAME=ddagent_integration
+    command: make -f DDMakefile install test_integration SUDO=sudo
+  '5.6': { <<: *base, image: 'circleci/php:5.6' }
+  '7.0': { <<: *base, image: 'circleci/php:7.0' }
+  '7.1': { <<: *base, image: 'circleci/php:7.1' }
+  redis_integration:
```

Comment:
[Minor] does the `_integration` suffix add any specification to this service? In other words, there will be any `redis` machine?
I would just write `redis` but this is just a personal preference for no apparent reason.

Reply:
My Idea was that `redis_integration` doesn't expose any ports. 
And then to have other `redis` to be used locally if need be.

I don't have a strong opinion about this. I'll change it to `redis`. And we'll see about exposing ports later

Reply:
Oh that makes sense. Please leave it here!